target/zephyr/Kconfig

```diff
@@ -95,36 +95,33 @@ if MENDER_MCU_CLIENT
 
     menu "Inventory configuration"
 
-        config MENDER_CLIENT_INVENTORY
-            bool "Mender client Inventory"
-            default y
+        config MENDER_CLIENT_INVENTORY_REFRESH_INTERVAL
+            int "Mender client Inventory refresh interval (seconds)"
+            range 0 86400
+            default 28800
             help
-                Inventory permits to send inventory key-value pairs to the Mender server.
-                It is particularly used to send artifact name and device type, and it permits to see the last check-in time of the device.
-
-        if MENDER_CLIENT_INVENTORY
+                Interval used to periodically send inventory to the Mender server.
+                Setting this value to 0 permits to disable the periodic execution and relies on the application to do it.
 
-            config MENDER_CLIENT_INVENTORY_REFRESH_INTERVAL
-                int "Mender client Inventory refresh interval (seconds)"
-                range 0 86400
-                default 28800
-                help
-                    Interval used to periodically send inventory to the Mender server.
-                    Setting this value to 0 permits to disable the periodic execution and relies on the application to do it.
-
-            config MENDER_CLIENT_INVENTORY_BUILD_INFO
-              bool "Build info built-in inventory"
-              default y
-              help
-                  Enable built-in build info inventory
+        config MENDER_CLIENT_INVENTORY_BUILD_INFO
+            bool "Build info built-in inventory"
+            default y
+            help
+                Enable built-in build info inventory
 
-            config MENDER_CLIENT_INVENTORY_NETWORK_INFO
-              bool "Network info built-in inventory"
-              default y
-              help
-                  Enable built-in network info inventory
+        config MENDER_CLIENT_INVENTORY_NETWORK_INFO
+            bool "Network info built-in inventory"
+            default y
+            help
+                Enable built-in network info inventory
 
-        endif
+        config MENDER_CLIENT_INVENTORY_DISABLE
+            bool "Disable Mender client Inventory (EXPERIMENTAL)"
+            default n
```

Comment:
Shouldn't all the above have `depend on MENDER_CLIENT_INVENTORY_DISABLE=n`?

Reply:
Technically yes, but it is harmless to let the user define them even when disabling inventory, and this being an experimental feature I didn't want to put it on top (which would make more sense) but then seems like we encourage it more.

If you insist I'll modify it.

Reply:
It can still be the last item and the ones above will disappear once the user selects it. They will see how much they are losing! Even more pressure! :wink:

Reply:
Modified